spec/converter_spec.rb

```diff
@@ -5,15 +5,14 @@
   include SEPA::Converter::InstanceMethods
 
   describe :convert_text do
-    it 'should convert special chars' do
+    it 'should correctly map chars' do
```

Comment:
Minor: perhaps the "correctly" could be something concrete, like "should map characters to the SEPA Requirements for an Extended Character Set".

Reply:
Yeap I agree! Fixed by [cf9c5f4](https://github.com/salesking/sepa_king/pull/105/commits/cf9c5f49f2936b8dfcbf38bc10f77a0f1ad7fa62)